src/course-checklist/ChecklistSection/ChecklistSection.test.jsx

```diff
@@ -1,255 +1,84 @@
-
-      const comment = screen.getByTestId('comment-section-assignmentDeadlines');
-      expect(comment.children).toHaveLength(0);
-    });
-
-    it('should have comment section', () => {
-      renderComponent(defaultProps);
-
-      const comment = screen.getByTestId('comment-section-assignmentDeadlines');
-      const assigmentLinks = within(comment).getAllByRole('link');
-
-      expect(comment.children).toHaveLength(1);
-
-      expect(screen.getByText(
-        messages.assignmentDeadlinesComment.defaultMessage,
-        { exact: false },
-      )).toBeVisible();
-
-      expect(assigmentLinks).toHaveLength(2);
-
-      expect(assigmentLinks[0].textContent).toEqual('Subsection');
-
-      expect(assigmentLinks[1].textContent).toEqual('ORA subsection');
-    });
-  });
-});    
-
-testChecklistData.forEach((check) => {
-  describe(`check with id '${check.id}'`, () => {
-    let checkItem;
-    beforeEach(() => {
-      initializeMockApp({
-        authenticatedUser: {
-          userId: 3,
-          username: 'abc123',
-          administrator: false,
-          roles: [],
-        },
-      });
-      store = initializeStore(initialState);
-      renderComponent(defaultProps);
-      checkItem = screen.getAllByTestId(`checklist-item-${check.id}`);
-    });
-
-    it('renders', () => {
-      expect(checkItem).toHaveLength(1);
-    });
-
-    it('has correct icon', () => {
-      const icon = screen.getAllByTestId(`icon-${check.id}`)
-
-      expect(icon).toHaveLength(1);
-
-      const { queryByTestId } = within(icon[0]);
-      if (completedItemIds.includes(check.id)) {
-        expect(queryByTestId('completed-icon')).not.toBeNull();
-      } else {
-        expect(queryByTestId('uncompleted-icon')).not.toBeNull();
-      }
-    });
-
-    it('has correct short description', () => {
-      const { getByText } = within(checkItem[0]);
-      const shortDescription = messages[`${check.id}ShortDescription`].defaultMessage;
-      expect(getByText(shortDescription)).toBeVisible();
-    });
-
-    it('has correct long description', () => {
-      const { getByText } = within(checkItem[0]);
-      const longDescription = messages[`${check.id}LongDescription`].defaultMessage;
-      expect(getByText(longDescription)).toBeVisible();
-    });
-
-    describe('has correct link', () => {
-      const links = getUpdateLinks('courseId')
-      const shouldShowLink = Object.keys(links).includes(check.id);
-
-      if (shouldShowLink) {
-        it('with a Hyperlink', () => {
-          const { getByRole, getByText } = within(checkItem[0]);
-
-          expect(getByText('Update')).toBeVisible();
-
-          expect(getByRole('link').href).toMatch(links[check.id]);
-        });
-      } else {
-        it('without a Hyperlink', () => {
-          const { queryByText } = within(checkItem[0]);
-
-          expect(queryByText('Update')).toBeNull();
-        });
-      }
-    });
+    expect(window.location.href).toBe('http://localhost/');
```

Comment:
[question] Should we hardcode `http://localhost/` instead of getting them from config?

Reply:
Corrected